components/drata/sources/failed-monitor/failed-monitor.mjs

```diff
@@ -0,0 +1,101 @@
+import drata from "../../drata.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/MonitorsPublicController_listMonitors/";
+
+export default {
+  key: "drata-failed-monitor",
+  name: "Failed Monitor",
+  description: `Emit new event whenever a monitor fails. [See docs here](${docsLink}).`,
+  type: "source",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    drata,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+    checkType: {
+      type: "string",
+      label: "Check Type",
+      description: "Filter monitors by check type (associated monitor instances)",
+      optional: true,
+      options: [
+        "POLICY",
+        "IN_DRATA",
+        "AGENT",
+        "INFRASTRUCTURE",
+        "VERSION_CONTROL",
+        "IDENTITY",
+        "TICKETING",
+        "HRIS",
+        "OBSERVABILITY",
+      ],
+    },
+  },
+  hooks: {
+    async deploy() {
+      const response = await this.drata.listMonitors({
+        paginate: true,
+        params: {
+          checkResultStatus: "FAILED",
+          reportInterval: "WEEKLY",
+        },
+      });
+
+      const visitedIds = {};
+      for (const monitor of response.data) {
+        const ts = Date.parse(monitor.lastCheck);
+        visitedIds[monitor.id] = ts;
+
+        this.$emit(monitor, {
+          id: `${monitor.id}_${ts}`,
+          summary: `Historical failed monitor event: ${monitor.name}`,
```

Comment:
```suggestion
          summary: `Historical failed monitor event: ${monitor.name}`,
```

Reply:
Changed 'Historical' to 'historical' to follow Google's Developer docs style guide."